scripts/check-dpdk.sh

```diff
@@ -9,10 +9,10 @@ fi
 cd $(readlink -e $(dirname $0))/..
 
 # Build ODP
-git clone https://github.com/OpenDataPlane/odp-dpdk --branch v1.41.0.0_DPDK_22.11 --depth 1
+git clone https://github.com/OpenDataPlane/odp-dpdk --branch v1.45.1.0_DPDK_22.11 --depth 1
 pushd odp-dpdk
 ./bootstrap
-./configure --prefix=$(pwd)/install --without-tests --without-examples
+./configure --prefix=$(pwd)/install --enable-dpdk-shared --without-tests --without-examples
```

Comment:
Why this change? Othewise this PR looks good to me.

Reply:
Static linking of dpdk became the default in odp-dpdk already a couple of years ago. This change is just a small convenience so that the script works with distro dpdk, which is probably shared only, so that there's no need to compile dpdk. If one does want to compile dpdk, then there's the devbuild script for that.